pkg/pipelinerunmetrics/metrics.go

```diff
@@ -421,8 +449,6 @@ func (r *Recorder) RunningPipelineRuns(lister listers.PipelineRunLister) error {
 	metrics.Record(ctx, runningPRsWaitingOnTaskResolutionCount.M(float64(trsWaitResolvingTaskRef)))
 	metrics.Record(ctx, runningPRsWaitingOnTaskResolution.M(float64(trsWaitResolvingTaskRef)))
 	metrics.Record(ctx, runningPRsCount.M(float64(runningPipelineRuns)))
-	metrics.Record(ctx, runningPRs.M(float64(runningPipelineRuns)))
```

Comment:
I think you are right.
currently  runningPRs and runningPRsCount offer same functionality of total running PRs count at cluster level.

Maybe we can deprecate one of them and  enhance the functionality to  provide data at namespace/pipeline/pipelinerun level.

I agree this is not backward compatible so for now we can introduce a new metric for  enhanced feature  and deprecate one of them.

WDYT ?

Reply:
One of the metrics is deprecated. Metrics having `count` doesn't satisfy the prometheus naming convention. We didn't remove it.

Default behaviour should be consistent with current behaviour. Only if user specify the config, we should add those tags.

Reply:
The reason for deprecation is that metrics doesn''t satisfy the naming convention.

Reply:
Of course, it's not just the name. Having `count` makes it look like it is being generated as part of a histogram. 

So some toolings fail sometimes. Already, raised a PR to remove these deprecated metrics.

Reply:
Done

Reply:
If I understood correct, `runningPRsCount` is deprecated because of its name, and should be removed, but not here, in a separate PR. The metric `runningPRs` instead should stay, and it's default behaviour should not change with this pull request.

Reply:
> If I understood correct, `runningPRsCount` is deprecated because of its name, and should be removed, but not here, in a separate PR. The metric `runningPRs` instead should stay, and it's default behaviour should not change with this pull request.

runningPRs is not getting deleted here.
this is moved  inside if block.